### PR TITLE
Add ability to change app language

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
Add the ability to change app language in Android settings for devices with Android 13 and upper

### CODE REVIEW

**Comment by meenbeese** [comment]
Thanks! I have some questions.

1. Which languages are available to choose from?
2. Why should `en-GB` be used, is it the default?
3. How does it improve current language selection?

File: app/src/main/res/resources.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+unqualifiedResLocale=en-GB
```
Comment (meenbeese) [inline app/src/main/res/resources.properties:1]: nit: add new line at the end

**Comment by Jean-BaptisteC** [comment]
1. All languages have translations in values directories, implementing this feature can reduce the size of app, because the compiler keeps library translations only for languages specified
2. Yes it's the default value
3. We can't change app language in app settings, it's very useful to test different languages without changing language of the device (for example to test RTL, if in the future we support Arabic language)

**Comment by meenbeese** [comment]
Thanks for the contribution. Maybe we can add/improve RTL support in the future ;)

**Pull request merged**